components/html/modules/content-patterns/_typography.html

```diff
@@ -3,79 +3,79 @@
-				<h2>H2 Archivo Narrow 32px #98700d</h2>
+				<h2>H2 Archivo Narrow 2em #98700d</h2>
 				<p>Explicabo, impedit repudiandae maxime.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
-				<h3>H3 Archivo Narrow Bold 28px #707070</h3>
+				<h3>H3 Archivo Narrow Bold 1.75em #707070</h3>
 				<p>Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit.  Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
-				<h4>H4 Archivo Bold 24px Caps #98700d</h4>
+				<h4>H4 Archivo Narrow Bold 1.5em Caps #98700d</h4>
 				<p>Dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime. <a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a> Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla.</p>
 				<aside class="highlight">
-					<h3 class="highlight__title">Archivo Bold 18px Caps #000000</h3>
+					<h3 class="highlight__title">Uses the H5 Style</h3>
 					<p class="highlight__content">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur ipsum perspiciatis veniam, rem totam optio distinctio. Porro, nisi in unde, nemo nulla totam corporis beatae natus, tempora hic aliquid. Voluptatum.</p>
 				</aside>
-				<h5>H5 Archivo Bold 22px Caps #000000</h5>
+				<h5>H5 Archivo Narrow Bold 1.375em Caps #000000</h5>
 				<p>Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
-				<h6>H6 Archivo Bold 18px Caps #000000</h6>
+				<h6>H6 Archivo Narrow Bold 1.125em Caps #000000</h6>
 				<p>Aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
 				<aside class="quote">
```

Comment:
Let's go ahead and apply the `.quote` style to the `blockquote` element as its base. We may need override some default bootstrap styles.